### ISSUE TITLE
Listen to radio station in the favorites

### ISSUE THREAD

**Issue by Dylantje** [jishi/node-sonos-http-api#790]
Dear..
Nice work you make.

Is there perhaps a option possible to play the favorite radio stations?
There a some onlin radio stations that i like to hear when i use the sonos.

Or do you now how to use that with your api?

Thanks!!!

**Comment by jsiegenthaler** [comment]
@Dylantje  Yes that is possible, I do it on my Sonos. Use the Sonos app, store your radio station as a favorite. Then you can play the favorite like this:
http://x.x.x.x.:5005/roomName/favorite/favoriteName

**Comment by Dylantje** [comment]
Thanks GREAT work again..
Is your api up to date?
Works fine with all the linux boxes/pi etc?
Or do i need to install a lot of things?


Is there a simple way to "read " the sonos..
http://x.x.x.x.:5005/roomName/favorite/favoriteName something like mqqt explorer?
Or are all the examples in the wiki?

**Comment by Dylantje** [comment]
Some error:
`[ General system configuration (beta): armbian-config ]

Last login: Sun Jan 24 21:13:10 2021 from 192.168.5.204
pi@odroidxu4:~$ npm install --production
npm WARN saveError ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN pi No description
npm WARN pi No repository field.
npm WARN pi No README data
npm WARN pi No license field.

audited 124 packages in 2.702s
found 2 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
pi@odroidxu4:~$
pi@odroidxu4:~$ npm audit fix
npm ERR! code EAUDITNOPJSON
npm ERR! audit No package.json found: Cannot audit a project without a package.json

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-01-26T20_22_36_826Z-debug.log
pi@odroidxu4:~$ npm audit
npm ERR! code EAUDITNOPJSON
npm ERR! audit No package.json found: Cannot audit a project without a package.json

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-01-26T20_22_46_609Z-debug.log
pi@odroidxu4:~$ npm install
npm WARN saveError ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN pi No description
npm WARN pi No repository field.
npm WARN pi No README data
npm WARN pi No license field.

audited 124 packages in 2.193s
found 2 high severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
pi@odroidxu4:~$ npm start
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /home/pi/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-01-26T20_23_11_333Z-debug.log
pi@odroidxu4:~$
`

**Comment by jsiegenthaler** [comment]
The api interface is described in the Readme.md file, you can see it on the github page for Sonos-http-api. 

You can view stored favourites with the .../roomName/favorite page

Check your pi is up to date before installing Sonos-http-api

Make sure you are installing from your default user, (normally called pi)

**Comment by jishi** [comment]
> Some error:
> `[ General system configuration (beta): armbian-config ]
> 
> Last login: Sun Jan 24 21:13:10 2021 from 192.168.5.204
> pi@odroidxu4:~$ npm install --production
> npm WARN saveError ENOENT: no such file or directory, open '/home/pi/package.json'
> npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
> npm WARN pi No description
> npm WARN pi No repository field.
> npm WARN pi No README data
> npm WARN pi No license field.
> 
> audited 124 packages in 2.702s
> found 2 high severity vulnerabilities
> run `npm audit fix` to fix them, or `npm audit` for details
> pi@odroidxu4:~$ pi@odroidxu4:~$ npm audit fix
> npm ERR! code EAUDITNOPJSON
> npm ERR! audit No package.json found: Cannot audit a project without a package.json
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR! /home/pi/.npm/_logs/2021-01-26T20_22_36_826Z-debug.log
> pi@odroidxu4:~$ npm audit
> npm ERR! code EAUDITNOPJSON
> npm ERR! audit No package.json found: Cannot audit a project without a package.json
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR! /home/pi/.npm/_logs/2021-01-26T20_22_46_609Z-debug.log
> pi@odroidxu4:~$ npm install
> npm WARN saveError ENOENT: no such file or directory, open '/home/pi/package.json'
> npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
> npm WARN pi No description
> npm WARN pi No repository field.
> npm WARN pi No README data
> npm WARN pi No license field.
> 
> audited 124 packages in 2.193s
> found 2 high severity vulnerabilities
> run `npm audit fix` to fix them, or `npm audit` for details
> pi@odroidxu4:~$ npm start
> npm ERR! code ENOENT
> npm ERR! syscall open
> npm ERR! path /home/pi/package.json
> npm ERR! errno -2
> npm ERR! enoent ENOENT: no such file or directory, open '/home/pi/package.json'
> npm ERR! enoent This is related to npm not being able to find a file.
> npm ERR! enoent
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR! /home/pi/.npm/_logs/2021-01-26T20_23_11_333Z-debug.log
> pi@odroidxu4:~$
> `

You seem to be running the commands in `/home/pi` but you probably have extracted or cloned the repo into a subdirectory.
